main.css

```diff
@@ -0,0 +1,261 @@
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: auto;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+	max-width: none;
+}
+
+html, body{
+	font-family: "Open Sans", sans-serif;
+	margin:auto;
+}
+
+
+/*********   header   **********/
+header {
+	background: #b30000;
+	text-align: center;
+	width: 100%;
+	color: #ffffff;
+	margin-bottom: 1px;
+}
+
+.header-element {
+	display: inline-block;
+	vertical-align: middle;
+	width: 30%;
+	margin: auto;
+	align-content: center;
+}
+
+h2 {
+	font-size: 1.5em;
+}
+
+#dropdown{
+	float: left;
+	display: none;
+}
+
+#hamburger {
+	float: left;
+}
+
+#hamburger img {
+	display: visible;
+	width: 30px;
+}
+
+#pokeball {
+	text-align: right;
+	visibility: hidden;
+}
+
+/*********   article   **********/
+article {
+
+}
+
+.search-box {
+	text-align: center;
+	width: 80%;
+}
+
+input[type=text] {
+	width: 100%;
+    padding: 12px 20px;
+    margin: 8px 0;
+    box-sizing: border-box;
+	font-size: 1em;
+}
+
+input {
+	margin: 15px;
+	width: 94%;
+}
+
+button {
+	width: 15%;
+	height: 60%;
+	margin-top: 10px;
+	margin-left: 50px;
+	border-radius: 25px;
+	font-size: 16px;
+}
+
+/*********   section   **********/
+section {
+	margin-bottom: 10px;
+}
+article {
+	background-image: url('assets/backgrounds/background_home.png');
+	/* background-size: cover; */
+	background-repeat: repeat;
+}
+
+.selected_poke_container {
+	width: 80%;
+	/* display: flex; */
+}
+
+#selected_poke_pic{
+	display: inline-block;
+	width: 100%;
+	float: left;
+	/* padding-bottom: 5px; */
+}
+
+.poke_pic img {
+	/* display: inline-block; */
+	width:90pt;
+	margin: 0 auto;
+	display: block;
+}
+
+.selected_poke_name {
+	background-color: rgb(94, 143, 204);
+	border: 1px solid black;
+	color: #FFFFFF;
+	float: left;
+	width: 100%;
+	padding: 5px 0px;
+	text-align: center;
+}
+
+#low {
+	visibility: hidden;
+}
+
+#selected_poke_list {
+	width: 100%;
+	float: left;
+	background-color: #FFFFFF;
+	border: 1px solid;
+	/* padding: 5px 40px; */
+	/* height: 100px; */
+	/* margin-top: 5px; */
+	/* font-size: 24px; */
+}
+
+#selected_poke_list ul {
+	padding: 5px 80px;
+
+}
+
+#poke_text {
+	display: inline-block;
+	margin: 20px auto;
+  border-radius: 25px;
+  border: 5px solid #b30000;
+  padding: 20px;
+	background: #ffffff;
+}
+
+.pokemon-pic {
+	text-align: center;
+}
+
+.pokemon-pic img {
+	display: inline-block;
+	width:90pt;
+	margin: 0 auto;
+}
+
+#pokemon-list {
+	width: 300px;
+	margin-bottom: 0;
+}
+
+.invisible {
+	display: none;
+}
+
+/*********   nav   **********/
+
+nav {
+	margin-top: 10px;
+	background: #538cc6;
+	vertical-align: middle;
+	margin: auto;
+	text-align: center;
+}
+
+.nav-button {
+	display: inline-block;
+	padding: 15px 5%;
+	color: white;
+	text-decoration: none;
+	cursor: pointer;
+}
+
+.nav-button:hover {
+	background: #80b3ff;
+}
+
+/*********   @media queries   **********/
+
+@media only screen and (min-width: 500px) {
```

Comment:
good concise media query code. however, i would suggest creating external css files and linking to them from your html instead.